pkg/services/accesscontrol/resourcepermissions/api.go

```diff
@@ -445,3 +447,19 @@ func permissionSetResponse(cmd setPermissionCommand) response.Response {
 	}
 	return response.Success(message)
 }
+
+func middlewareUserUIDResolver(userService user.Service, paramName string) web.Handler {
```

Comment:
middleware is duplicated here as following the pattern of putting the middleware in the service causes context handler circular dependencies.

Root logic is still shared